### DISCUSSION TITLE
时间同步概述

### DISCUSSION THREAD

**Discussion by giscus** [caibingcheng/comment#317] [Announcements]
# 时间同步概述

本文涉及的时间同步协议包括：NTP、PTP。泛泛而谈，以构建对时间同步的基本认识。
当我第一次接触时间同步的概念时，是很疑惑的。为什么要做时间同步？时间同步是什么？为什么会有各种各样的协议？本文将尽量以我的认识从几个方面解答：

什么是时间同步；
为什么要做时间同步；
时间同步协议的基本原理；

1和2是困扰我比较久的问题，3则是可以通过代码和文档直接获得的知识，没有太多的阻塞点。

https://imcbc.cn/202406/time-sync-summary/

**Comment by penndu** [comment]
chrony了解下~
